### PR TITLE
use pagination repository interface

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by nicolasThal** [comment]
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/467
https://github.com/open-orchestra/open-orchestra-model-interface/pull/131
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/6
https://github.com/open-orchestra/open-orchestra-libs/pull/33
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1226
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/62
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/84

File: Pagination/Configuration/FilterRepositoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+/**
+ * Interface FilterRepositoryInterface
+ */
+interface FilterRepositoryInterface
+{
+    /**
+     * @param FilterTypeManager $filterTypeManager
+     */
+    public function setFilterTypeManager(FilterTypeManager $filterTypeManager);
+}
```
Comment (itkg-alavieille) [inline Pagination/Configuration/FilterRepositoryInterface.php]: eof

File: Pagination/Configuration/PaginationRepositoryInterface.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    public function count();
+
+    /**
+     * @param FinderConfiguration $configuration
+     *
+     * @return mixed
+     */
+    public function countWithFilter(FinderConfiguration $configuration);
+
+}
```
Comment (itkg-alavieille) [inline Pagination/Configuration/PaginationRepositoryInterface.php]: eof

**Comment by nicolasThal** [comment]
@itkg-alavieille pr updated

**Comment by ghost** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

**Pull request merged**